src/core/compose.nix

```diff
@@ -134,6 +134,10 @@ let
               _if = __isStd__;
               std = l.removeAttrs (extern // atom) [ "std" ];
             }
+            {
+              _if = !__isStd__;
+              use = l.removeAttrs extern [ "std" ];
```

Comment:
hmm, now I wonder if we should, for the sake of consistency, place `std` under `use`. So instead of accessing `std` from the top-level, we would use `use.std`?